src/avram/paramable.cr

```diff
@@ -3,4 +3,8 @@ module Avram::Paramable
   abstract def nested(key : String) : Hash(String, String)
   abstract def get?(key : String)
   abstract def get(key : String)
+
+  def has_key_for?(operation : Avram::Operation.class | Avram::SaveOperation.class) : Bool
+    nested?(operation.param_key).any?
```

Comment:
Since Avram::Params are mainly used to circumvent some testing stuff, the actual specs for this will go in Lucky itself.

Reply:
I think specs should definitely be here so we know we don't break Avram. 

I think we can still test this by making a fake class that implements this `Paramable` and then tests that it returns the right thing. 

See example here: https://github.com/luckyframework/avram/blob/79668a93943c7706bb775c01b5fd34d02c342c3f/spec/operations/nested_save_operation_spec.cr#L17-L58